src/components/formfields/saveCancel/__tests__/SaveCancel-test.js

```diff
@@ -0,0 +1,94 @@
+import { shallow } from 'enzyme';
+import { shallowToJson } from 'enzyme-to-json';
+import React from 'react';
+import SaveCancel from '../SaveCancel';
+import sinon from 'sinon';
+import ReactTestUtils from 'react-addons-test-utils';
+
+
+describe('SaveCancel snapshot', () => {
+  const onClickSave = sinon.spy();
+
+  it('should match snapshot', () => {
+    const wrapper = shallow(
+      <SaveCancel
+        id="2"
+        saveLabel="Hii"
+        onClickSave={onClickSave}
+        cancelDisabled={true}
+      />
+    );
+
+    expect(wrapper.find('.cancelButton').everyWhere(n => n.hasClass('cancelButton'))).toEqual(true);
```

Comment:
Remove all these wrapper.find('.XXXButton').[every|some|has].. they are absolutely usesless.

What i wanted you to do with a class on the button was:

```wrapper.find('.submitButton').simulate('click');```
and then:
```expect(someSubmitButtonSpy.calledOnce).toBe(true)```
thats it.

Other than that, a snapshot test is good enoug. 

You dont need wrapper.debug or wrapper.html tests